app/src/main/java/org/simple/clinic/summary/linkId/LinkIdWithPatientView.kt

```diff
@@ -185,7 +168,7 @@ class LinkIdWithPatientView :
   private fun viewShows(): Observable<UiEvent> {
     return downstreamUiEvents
         .ofType<LinkIdWithPatientViewShown>()
-        .map { it as UiEvent }
+        .map { it }
```

Comment:
Is this necessary anymore since we are not actually converting it to anything else?

Reply:
Please disregard this since this event stream has been removed in a later commit.